src/webgl/light.js

```diff
@@ -592,13 +592,63 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
  * @alt
  * light with slider having a slider for varying roughness
  */
-p5.prototype.imageLight = function(img){
+p5.prototype.imageLight = function (img) {
   // activeImageLight property is checked by _setFillUniforms
   // for sending uniforms to the fillshader
   this._renderer.activeImageLight = img;
   this._renderer._enableLighting = true;
 };
 
+/**
+ * Creates a Panorama with given image.
```

Comment:
We can probably take this line out if we move that last paragraph up to the top, it serves as a pretty good intro.